pkgs/top-level/all-packages.nix

```diff
@@ -41685,7 +41685,7 @@ with pkgs;
 
   snowsql = callPackage ../applications/misc/snowsql { };
 
-  snowmachine = python3Packages.callPackage ../applications/misc/snowmachine { };
+  snowmachine = python3Packages.callPackage ../by-name/sn/snowmachine/package.nix { };
```

Comment:
snowmachine is an application and should be using `buildPythonApplication` instead of `buildPythonPackage`

also you can avoid entry in all-packages, by calling `python3Packages.buildPythonApplication` directly

Reply:
What do you mean by calling it directly?

Reply:
yeah, I updated to using that so looks like I don't need the entry anymore so I've removed it from all-packages now as well